frontend/pages/admin/pacta-version/index.vue

```diff
@@ -0,0 +1,56 @@
+<script setup lang="ts">
+import { type PactaVersion } from '@/openapi/generated/pacta'
+
+const router = useRouter()
+const { pactaClient } = useAPI()
+const { error: { withLoadingAndErrorHandling, handleOAPIError } } = useModal()
+
+const prefix = 'admin/pacta-version'
+const pactaVersions = useState<PactaVersion[]>(`${prefix}.pactaVersions`, () => [])
+
+const deletePV = (id: string) => withLoadingAndErrorHandling(
+  () => pactaClient.deletePactaVersion(id)
+    .then(handleOAPIError)
```

Comment:
This is a great utility, though I think with a bit of JavaScript trickery, you could create a new type that exposes all the same functions as the `pactaClient`, but calls handleOAPIError on the result. Alternatively, we can reconfigure our API definitions or TS codegen to throw an error on non-200s and catch those as is.

If I'm understanding correctly, we'll probably want `handleOAPIError` on every single `pactaClient.whatever` call (unless something wants to handle the error explicitly), so I'd like to have something more bulletproof, as we'll invariably forget it.